src/packagers/JSConcatPackager.js

```diff
@@ -88,19 +91,34 @@ class JSConcatPackager extends Packager {
     for (let identifier in asset.cacheData.imports) {
       let [source, name] = asset.cacheData.imports[identifier];
       let dep = asset.depAssets.get(asset.dependencies.get(source));
+
+      if (name === '*') {
+        this.markUsedExports(dep);
+      }
+
       this.markUsed(dep, name);
```

Comment:
- `a.js` : 
```js
import * as b from './b'

console.log(b)
```

- `b.js` : <== `b.js` should be `dep` here, if we remove `markUsed` it'll get trimmed because it doesn't have any exports/import in the `cacheData` (just 2 elements in `wildcards`). This will make
```js
export * from './c'
export * from './d'
```

Using `*` seemed like a nice fix to me for this, as it'll keep `usedImports.size > 0` and never conflict with existing import. I'm just getting familiar with this part though, so not really sure